templates/SilverStripe/Admin/LeftAndMain.ss

```diff
@@ -1,20 +1,29 @@
 <!DOCTYPE html>
 <html lang="$Locale.RFC1766">
-	<head>
-	<% base_tag %>
-	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-	<title>$Title</title>
+    <head>
+    <% base_tag %>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>$Title</title>
 </head>
 <body class="loading cms" data-frameworkpath="$ModulePath(silverstripe/framework)"
-	data-member-tempid="$CurrentMember.TempIDHash.ATT"
+    data-member-tempid="$CurrentMember.TempIDHash.ATT"
 >
-	<% include SilverStripe\\Admin\\CMSLoadingScreen %>
+    <% include SilverStripe\\Admin\\CMSLoadingScreen %>
 
-	<div class="cms-container" data-layout-type="custom">
-		$Menu
-		$Content
-		$PreviewPanel
+    <a class="cms-container-skip-link"
+       href="#cms-container-skip-link-target"
+       tabindex="0"
+    ><%t SilverStripe\\Admin\\LeftAndMain.SkipLink "Skip to main content" %></a>
```

Comment:
I think "Skip main navigation" would be more appropriate - "main content" is really more of a front-end concept IMO. The CMS doesn't have "content" in the same way - it just has a bunch of forms for editing content.

Take `/admin/pages` for example. If you select "skip to main content" it feels a bit weird to be confronted by another menu, in which you have to select a page before seeing [an edit form containing] any actual content.

Reply:
Updated